tasks/connectors/ms_defender_tvm/ms_defender_tvm.rb

```diff
@@ -83,7 +83,6 @@ def self.metadata
 
       def run(opts)
```

Comment:
It would be even better to split the run method into smaller methods. But it’s more about code style and readability.

Reply:
same this code too. planning to leave this as is